db/schema.rb

```diff
@@ -239,6 +241,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "temp_enrollments_from_csv", id: false, force: :cascade do |t|
+    t.bigint "member_id"
+    t.bigint "mission_id"
+    t.bigint "id"
+    t.time "old_start_time"
+    t.time "old_end_time"
+    t.datetime "start_time"
+    t.datetime "end_time"
+  end
+
```

Comment:
I guess this table comes from another ticket, does it?

Reply:
yes it come from https://github.com/Mate2xo/Creeons_la_coop/tree/restore_enrollments_with_csv
it is removed.